src/helpers/find_sorted_positions.rs

```diff
@@ -0,0 +1,253 @@
+use crate::modulus_convert::RandomShareGenerationHelper;
+use crate::modulus_convert::ReplicatedFp31SecretSharing;
+
+// Helper functions
+#[must_use]
+pub fn gen_mult_input(
+    shares: &[ReplicatedFp31SecretSharing],
+    helper: &RandomShareGenerationHelper,
+) -> Vec<(ReplicatedFp31SecretSharing, ReplicatedFp31SecretSharing)> {
+    // Step1: Calculate x and 1 - x
+    let mut x_and_one_minus_x: Vec<ReplicatedFp31SecretSharing> = shares
```

Comment:
i think it's more readable to use:

```rust
let mut x_and_one_minus_x = shares
    .iter()
    .map(|x| helper.get_share_of_one() - *x)
    .collect::<Vec<_>>();
```
so that the `collect` and the type are right next to each other